src/main/java/uk/gov/hmcts/reform/document/DocumentUploadClientApi.java

```diff
@@ -47,14 +50,34 @@ public DocumentUploadClientApi(
         this.objectMapper = objectMapper;
     }
 
-    public UploadResponse upload(
+    public UploadResponse uploadWithUserId(
```

Comment:
suggestion: may be we can just overload the method instead of having diff method names because userId is always required.

Reply:
done